configs/common/packages.yaml

```diff
@@ -37,7 +37,7 @@ packages:
   ecflow:
```

Comment:
I think you want to also specify the bufr_query version here after bufr (line 23) with the same require syntax

Reply:
There's no need to retest, just add the two lines please and we can merge the PR. Thanks.

Reply:
@climbfuji thanks for the feedback. I've added the common/packages.yaml entry for bufr-query.